uwblueprint/sistering
# User Profile Panel

## Ticket link

Closes #401 and #403



## Implementation description
* Created the top and bottom section of the user profile panel, incorporated @neelismail01's middle section into the panel as well 
* Updated hardcoded data in AdminUserManagementPage (and prop types) to match the side panel implementation (updating the query isn't included in this ticket) 



## Steps to test
1. Navigate to localhost:3000/admin/users
<img width="1177" alt="Screen Shot 2022-07-25 at 10 54 32 PM" src="https://user-images.githubusercontent.com/82241706/180913100-9edf31db-0143-4447-9625-5abfd1b423c0.png">
2. Click on the 3 dots beside on the "Amanda Du 1" row (sample volunteer user side panel), verify it displays the following 
<img width="1482" alt="Screen Shot 2022-07-25 at 10 55 34 PM" src="https://user-images.githubusercontent.com/82241706/180913290-dd040d64-b449-4ca6-875b-6f029e03d99b.png">
3. Click on the 3 dots beside on the "Amanda Du 1" row (sample admin user side panel), verify it displays the following
<img width="1480" alt="Screen Shot 2022-07-25 at 10 55 47 PM" src="https://user-images.githubusercontent.com/82241706/180913408-880d4a39-fb3c-4173-8c26-c0ba96847c2f.png">
4. Click on the "remove user" button should open a modal (modal logic implementation is part of another ticket) 


## What should reviewers focus on?
* Ensure the implementation matches the design and specified requirements on figma (https://www.figma.com/file/TWJTNWjK75A6VgSisySTLe/Sistering-%2F-Admin-Experience?node-id=1643%3A48828)
* Ensure @neelismail01's MultiBranchSelector component integrates correctly with this side panel

## Checklist
